### ISSUE TITLE
Update wdio tests to comply with new Axe accessibility rule

### ISSUE THREAD

**Issue by kschuste** [cerner/terra-core#1113]
# Issue Description
New axe accessibility rule can cause wdio tests to fail for some components.  Every page should have a landmark defined on the page.  Is described here:  https://dequeuniversity.com/rules/axe/3.0/landmark-one-main.

Will need to either update the tests to have a landmark defined or potentially turn off the rule when running the wdio tests.

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [ ] Bug
- [X] Other

## Expected Behavior
All tests should pass and the build should succeed.

**Comment by kschuste** [comment]
<img width="1788" alt="screen shot 2018-01-02 at 2 51 09 pm" src="https://user-images.githubusercontent.com/1711642/34499606-69c69a26-efcc-11e7-8ace-6637f1d212b9.png">

**Comment by emilyrohrbough** [comment]
This was resolved in https://github.com/cerner/terra-core/pull/1152
